server/hotrod/src/main/java/org/infinispan/server/hotrod/util/BulkUtil.java

```diff
@@ -21,17 +17,18 @@
    public static final int LOCAL_SCOPE = 2;
 
    public static Iterator<byte[]> getAllKeys(Cache<byte[], ?> cache, int scope) {
```

Comment:
I left this class since or for some reason the ```scope``` parameters is not used, and it's part of the Hot Rod protocol. Need to check what's going on here